.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# Issue Title
```

Comment:
Consider two template or better splitting this up between issue and PR. Internal links at the top work.